content/app/guides/multi-app-solution/preparations/_index.en.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Preparations before making a multi-app solution in Altinn
+linktitle: Preparations
+description: What preparations that should be done before creating a multi-app solution
+weight: 10
+toc: true
+aliases:
+
+- /app/multi-app-solution/preparations/
+
+---
+
+A crucial part of the multi-app solution is to make sure the
```

Comment:
focus on the prerequisites here instead of describing how the solution works. A large portion of this text should be covered by other sections I would say. An illustration to accompany the general description might also be beneficial